fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -383,10 +397,55 @@ private CapabilityStatement buildCapabilityStatement() throws Exception {
         return conformance;
     }
 
+    private List<Rest.Resource.Interaction> buildInteracations(List<String> interactionConfig) throws FHIRPersistenceException {
+        if (interactionConfig == null) return null;
+
+        List<Rest.Resource.Interaction> defaultInteractions = new ArrayList<>();
+        for (String interactionString : interactionConfig) {
+            if (interactionString.equals("search")) {
```

Comment:
Personal preference.  "search".equals(interactionString)
Same for other else if checks in this method.

Reply:
suggestion taken